test/runtests.jl

```diff
@@ -0,0 +1,14 @@
+using Test
+
+include(joinpath(@__DIR__, "test_tags.jl"))
```

Comment:
```suggestion
include( "test_tags.jl")
```

Include (when used at toplevel) is alway a relative path